### PR TITLE
fix #2047: localize micro survey prompts & responses

### LINKED ISSUE #2047: localize microsurvey questions & answers
https://github.com/mozilla/blurts-server/pull/2043 added new microsurvey questions & answers for SUPR-Q and Product Market Fit. We should localize these and show them to all users.

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: locales/en/app.ftl [hunk 1]
```diff
@@ -163,6 +163,65 @@ fxa-account = {-brand-fxa}
...
+micro-survey-pmf-prompt = How would you feel if you could no longer use {-product-short-name}?
+
+# This is a survey response that follows the pmf prompt
+micro-survey-very-disappointed-response = Very disappointed
+
+# This is a survey response that follows the pmf prompt
+micro-survey-somewhat-disappointed-response = Somewhat disappointed
+
+# This is a survey response that follows the pmf prompt
+micro-survey-dont-care-response = I wouldn't care
```
Comment (flodolo) [inline locales/en/app.ftl:223]: `'`-> `’`

**Review by flodolo — APPROVED** [review]

File: template-helpers/hbs-helpers.js [hunk 2]
```diff
@@ -50,56 +50,75 @@ function microsurveyBanner(args) {
...
+    "micro-survey-dont-care-response",
+    "micro-survey-not-likely-response",
+    "micro-survey-very-likely-response",
+  ];
+  const enLocaleArgs = {
+    hash: args.hash, data: {root: {req: {supportedLocales: ["en"]} } },
+  };
+  let bannerOpeningDivDataset = "";
+  microSurveyResponseIds.forEach(id => {
+    bannerOpeningDivDataset += `data-${id}-translated="${getString(id, args)}"`;
```
Comment (pdehaan) [inline template-helpers/hbs-helpers.js:75]: Q: Does this need a space between the `"` and ``` ` ``` (like below)?

Comment (groovecoder) [reply template-helpers/hbs-helpers.js:75]: I'll add a space around each attribute just in case we remove or re-order one of these lines later.

**Comment by groovecoder** [comment]
Updated to address comments.

File: template-helpers/hbs-helpers.js [hunk 3]
```diff
@@ -50,56 +50,75 @@ function microsurveyBanner(args) {
...
+  const enLocaleArgs = {
+    hash: args.hash, data: {root: {req: {supportedLocales: ["en"]} } },
+  };
+  let bannerOpeningDivDataset = "";
+  microSurveyResponseIds.forEach(id => {
+    bannerOpeningDivDataset += ` data-${id}-translated="${getString(id, args)}" `;
+    bannerOpeningDivDataset += ` data-${id}-english="${getString(id, enLocaleArgs)}" `;
+  });
+  const bannerOpeningDiv = `<div id="micro-survey-banner" class="micro-survey-banner hidden" ${bannerOpeningDivDataset}>`;
   const nowSecond = Math.abs(Math.floor(new Date().getTime() / 1000)) % 10;
```
Comment (pdehaan) [inline template-helpers/hbs-helpers.js:79]: Q: Should this just be `new Date().getSeconds() % 10`?

Comment (groovecoder) [reply template-helpers/hbs-helpers.js]: Updated

File: locales/en/app.ftl (hunk 1 above)
Comment (groovecoder) [reply locales/en/app.ftl]: Updated.

**Review by pdehaan — APPROVED** [review]

**Review by maxxcrawford — APPROVED** [review]
LGTM 🚢

**Pull request merged**